docs/rules/unicode-bom.md

```diff
@@ -0,0 +1,63 @@
+# Allow or disallow the Unicode Byte Order Mark (BOM) (unicode-bom)
```

Comment:
Similarly I would suggest that the docs should say "Require or disallow" (allowing without requiring is done by not using the rule). Thanks!